windows/CommonSettings.props

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <BuildDir>$(SolutionDir)..\Build</BuildDir>
+    <!--NOTE: CpuOnlyBuild and UseCuDNN flags can't be set at the same time.-->
+    <CpuOnlyBuild>false</CpuOnlyBuild>
+    <UseCuDNN>true</UseCuDNN>
+    <CudaVersion>7.5</CudaVersion>
+    <!-- NOTE: If Python support is enabled, PythonDir (below) needs to be
+      set to the root of your Python installation. If your Python installation
+      does not contain debug libraries, debug build will not work. -->
+    <PythonSupport>false</PythonSupport>
+    <CudaDependencies></CudaDependencies>
+    <CudaArchitecture>compute_35,sm_35</CudaArchitecture>
```

Comment:
Then it's questionable why `compute_35` is the default. It is neither generic nor optimal. Many serious DNN users run on Maxwell GPUs nowadays.

This is another reason why making this an example file necessary. Users should be mindful about these settings.

(Btw, extra 10 mins in build time is nothing compared to the perf gain in run time.)